app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -41,8 +43,14 @@ public void attach(long eventId, IAttendeesView attendeesView) {
     }
 
     @Override
-    public void start() {
-        loadAttendees(false);
+    public void start(boolean forceStart) {
+        if (forceStart)
+            progress = 0;
+        if (progress == 0) {
+            loadAttendees(false);
+        } else {
+            attendeesView.showAttendees(attendeeList);
+        }
```

Comment:
IMO this is the problem of view. In view, we could check that if it is rotated and loading again, we'll just get the attendees, and if it is not, then we'll loaf